scripts/install_third_party_libs.py

```diff
@@ -125,9 +125,19 @@ def pip_install(package, version, install_path):
 
     # The call to python -m is used to ensure that Python and Pip versions are
     # compatible.
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', '%s==%s' % (package, version),
-        '--target', install_path])
+    try:
+        fnull = python_utils.open_file(os.devnull, 'w')
```

Comment:
Instead of doing this, please use `Popen` instead

Reply:
And then use communicate to grab the stdout and stderr.

Reply:
Thanks!
I used popen and got the error message
PTAL.